Pipfile

```diff
@@ -26,3 +26,4 @@ powersimdata = "~=0.4.4"
 nrel-pysam = "~=2.2"
 pyproj = "~=3.0"
 pygrib = "~=2.1.4"
+line-profiler = "*"
```

Comment:
Do we want to add this to the Pipfile. It seems that it could be installed by the user in his environment if needed.

Reply:
I think we can potentially take it out as a dependency, we mainly just need it for profiling